src/Geomap.js

```diff
@@ -241,6 +241,8 @@ export default class Geomap extends Viz {
     const r = scales[`scale${this._pointSizeScale.charAt(0).toUpperCase()}${this._pointSizeScale.slice(1)}`]()
       .domain(extent(pointData, (d, i) => this._pointSize(d, i)))
       .range([this._pointSizeMin, this._pointSizeMax]);
+    
+    if (this._fitObject || this._fitFilter || this._fitKey || this._topojson || this._topojsonFilter || this._topojsonKey) this._zoomSet = false;
```

Comment:
This logic will always reset the zoomSet if any of those methods are set... which is not the desired behavior. We want to reset the zoomSet if the user has CHANGED one of these methods. You need to add this reset directly into each one of these methods where the value gets assigned. For example:

```js
topojson(_, f) {
     if (arguments.length) {
       const prev = this._queue.find(q => q[3] === "topojson");
       const d = [load.bind(this), _, f, "topojson"];
       if (prev) this._queue[this._queue.indexOf(prev)] = d;
       else this._queue.push(d);
       this._zoomSet = false;
       return this;
     }
     return this._topojson;
   }
```

Reply:
Thanks for your feedback, teaching and orienting me about d3plus. I've seen improves in my ability of to solve bugs related with the library :)